O365/message.py

```diff
@@ -756,6 +762,24 @@ def reply(self, to_all=True):
 
         message = response.json()
 
+        # adding new feacture to the timezone 
+        html_content = message['body']['content']
+
+        pattern = re.compile(r'<b>Sent:</b> (.*?)<br>')
+
+        def change_sent_time(match):
+            sent_time_str = match.group(1)
+            sent_time = datetime.strptime(sent_time_str, '%A, %B %d, %Y %I:%M:%S %p')
+            # asign the time zone for example bogota
+            timezone_tz = pytz.timezone('America/Bogota')
```

Comment:
We shouldn't be hard-coding a timezone, this needs to be a variable
Also I'm pretty sure `pytz` has been dropped due to `zoneinfo` requirements: https://github.com/O365/python-o365/blob/master/CHANGES.md#version-2030-2023-09-27

Reply:
Exactly, no timezone hardcoded, no pytz usage